js/co-authors-plus.js

```diff
@@ -1,533 +1,504 @@
+				// Create new author-suggest and append it to a new row
+				var newCO = coauthors_create_autosuggest( '', false );
+				coauthors_add_to_table( newCO );
+				move_loading( newCO );
+			}
+		}
+
+		co.bind( 'blur', coauthors_stop_editing );
+
+		// Set the value for the auto-suggest box to the Author's name and hide it
+		co.val( unescape( author.name ) )
+			.hide()
+			.unbind( 'focus' );
+
+		return true;
+	}
+
+	/*
+	 * Add the autosuggest box and text tag to the Co-Authors table
+	 * @param object Autosuggest input box
+	 * @param object Text tag
+	 * @param
+	 */
+	function coauthors_add_to_table( co, tag, input, options ) {
+		if (co) {
+			var $div = jQuery( '<div/>' )
+						.addClass( 'suggest' )
+						.addClass( 'coauthor-row' )
+						.append( co )
+						.append( tag )
+						.append( input );
+
+			//Add buttons to row
+			if (tag) { coauthors_insert_author_edit_cells( $div, options ); }
+
+			jQuery( '#coauthors-list' ).append( $div );
+		}
+	}
+
+	/*
+	 * Adds a delete and edit button next to an author
+	 * @param object The row to which the new author should be added
+	 */
+	function coauthors_insert_author_edit_cells($div, options){
+
+		var $options = jQuery( '<div/>' )
+			.addClass( 'coauthors-author-options' );
+
+		/*
+        if(options.addEdit) {
+            var editBtn = jQuery('<span></span>')
+                            .addClass('edit-coauthor')
+                            .text(coAuthorsPlusStrings.edit_label)
+                            .bind('click', coauthors_edit_onclick)
+                            ;
+            td.append(editBtn);
+        }
+		*/
+		if (options.addDelete) {
+			var deleteBtn = jQuery( '<span/>' )
+								.addClass( 'delete-coauthor' )
+								.text( coAuthorsPlusStrings.delete_label )
+								.bind( 'click', coauthors_delete_onclick );
+			$options.append( deleteBtn );
+		}
+
+		$div.append( $options );
+		return $div;
+	}
+
+	/*
+	 * Creates autosuggest input box
+	 * @param string [optional] Name of the author
+	 * @param string [optional] Name to be applied to the input box
+	 */
+	function coauthors_create_autosuggest(authorName, inputName) {
+
+		if ( ! inputName) { inputName = 'coauthorsinput[]'; }
+
+		var $co = jQuery( '<input/>' );
+
+		$co.attr({
+			'class': 'coauthor-suggest'
+			, 'name': inputName
+			})
+			.appendTo( $coauthors_div )
+			/*
+            .autocomplete(coauthors_all, {
+                matchContains: true
+                , scroll: false
+                , formatItem: function(row) { return row[2] + ' ' + row[0] + ' | ' + row[1] }
+                , formatResult: function(row) { return row[1]; }
+            })
+			*/
+			.suggest(coAuthorsPlus_ajax_suggest_link, {
+				onSelect: coauthors_autosuggest_select
+			})
+			.keydown( coauthors_autosuggest_keydown );
+
+			if (authorName) {
+				$co.attr( 'value', unescape( authorName ) ); } else {
+				$co.attr( 'value', coAuthorsPlusStrings.search_box_text )
+				.focus( function(){ $co.val( '' ) } } )
```

Comment:
There's some weirdness with the automated cleanup. For example, There's now an extra `}` here, which will probably break things.

Before:

```
.focus( function(){ $co.val( '' ) } )
```

After:

```
.focus( function(){ $co.val( '' ) } } )
```

There may be other similar issues. Might be better to approach this cleanup in smaller chunks?